It was compiling before. And yes it is due to the modified code:
```
/afs/cern.ch/work/j/jsauvan/Projects/L1CalorimeterTrigger_Phase2HGCal/CMSSW/integrations/CMSSW_11_0_X_2019-06-24-2300/src/L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorCoarsenerImpl.cc: In member function 'void HGCalConcentratorCoarsenerImpl::coarsen(const std::vector<l1t::HGCalTriggerCell>&, std::vector<l1t::HGCalTriggerCell>&)':
/afs/cern.ch/work/j/jsauvan/Projects/L1CalorimeterTrigger_Phase2HGCal/CMSSW/integrations/CMSSW_11_0_X_2019-06-24-2300/src/L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorCoarsenerImpl.cc:40:19: error: 'kScintillatorPseudoThicknessIndex_' was not declared in this scope
       thickness = kScintillatorPseudoThicknessIndex_;
```
`kScintillatorPseudoThicknessIndex_` has not been defined in the scope of `HGCalConcentratorCoarsenerImpl`

--- a/L1Trigger/L1THGCal/interface/HGCalCoarseTriggerCellMapping.h
+++ b/L1Trigger/L1THGCal/interface/HGCalCoarseTriggerCellMapping.h
@@ -58,6 +58,10 @@ private:
   static constexpr int kHGCalCellMaskV9Inv_ = ~0xff;
   static constexpr int kHGCalScinCellMaskInv_ = ~0x1ffff;
 
+  static constexpr int kRoc0deg_ = 1;
+  static constexpr int kRoc120deg_ = 2;
+  static constexpr int kRoc240deg_ = 3;
+
   HGCalTriggerTools triggerTools_;
   HGCSiliconDetIdToROC detIdToROC_;
   std::vector<unsigned> ctcSize_;

--- a/L1Trigger/L1THGCal/interface/concentrator/HGCalConcentratorCoarsenerImpl.h
+++ b/L1Trigger/L1THGCal/interface/concentrator/HGCalConcentratorCoarsenerImpl.h
@@ -21,6 +21,7 @@ private:
   bool fixedDataSizePerHGCROC_;
   HGCalCoarseTriggerCellMapping coarseTCmapping_;
   static constexpr int kHighDensityThickness_ = 0;
+  static constexpr unsigned kScintillatorPseudoThicknessIndex_ = 3;
 
   struct CoarseTC {
     float sumPt;

--- a/L1Trigger/L1THGCal/interface/concentrator/HGCalConcentratorSuperTriggerCellImpl.h
+++ b/L1Trigger/L1THGCal/interface/concentrator/HGCalConcentratorSuperTriggerCellImpl.h
@@ -42,7 +42,7 @@ private:
 
   //Parameters for energyDivisionType_ = equalShare
   static constexpr int kTriggerCellsForDivision_ = 4;
-
+  static constexpr unsigned kScintillatorPseudoThicknessIndex_ = 3;
   class SuperTriggerCell {
   private:
     float sumPt_, sumMipPt_, maxMipPt_, fracsum_;

--- a/L1Trigger/L1THGCal/src/HGCalCoarseTriggerCellMapping.cc
+++ b/L1Trigger/L1THGCal/src/HGCalCoarseTriggerCellMapping.cc
@@ -68,15 +68,15 @@ uint32_t HGCalCoarseTriggerCellMapping::getRepresentativeDetId(uint32_t tcid) co
       int newV = 0;
       HGCalTriggerDetId tc_IdV9(tcid);
       int rocnum = detIdToROC_.getROCNumber(tc_IdV9.triggerCellU(), tc_IdV9.triggerCellV(), 1);
-      if (rocnum == 1) {
+      if (rocnum == kRoc0deg_) {
         uPrime = tc_IdV9.triggerCellU();
         newU = (uPrime & ~1);
         newV = tc_IdV9.triggerCellV() - tc_IdV9.triggerCellU() + newU;
-      } else if (rocnum == 2) {
+      } else if (rocnum == kRoc120deg_) {
         uPrime = tc_IdV9.triggerCellU() - tc_IdV9.triggerCellV() - 1;
         newU = (uPrime & ~1) + tc_IdV9.triggerCellV() + 1;
         newV = tc_IdV9.triggerCellV();
-      } else if (rocnum == 3) {
+      } else if (rocnum == kRoc240deg_) {
         uPrime = tc_IdV9.triggerCellV() - kRotate4_;
         newU = tc_IdV9.triggerCellU();
         newV = (uPrime & ~1) + kRotate4_;
@@ -136,15 +136,15 @@ uint32_t HGCalCoarseTriggerCellMapping::getCoarseTriggerCellId(uint32_t detid) c
       int vPrime = 0;
       int rocnum = detIdToROC_.getROCNumber(tc_IdV9.triggerCellU(), tc_IdV9.triggerCellV(), 1);
 
-      if (rocnum == 1) {
+      if (rocnum == kRoc0deg_) {
         uPrime = tc_IdV9.triggerCellU();
         vPrime = tc_IdV9.triggerCellV() - tc_IdV9.triggerCellU();
 
-      } else if (rocnum == 2) {
+      } else if (rocnum == kRoc120deg_) {
         uPrime = tc_IdV9.triggerCellU() - tc_IdV9.triggerCellV() - 1;
         vPrime = tc_IdV9.triggerCellV();
 
-      } else if (rocnum == 3) {
+      } else if (rocnum == kRoc240deg_) {
         uPrime = tc_IdV9.triggerCellV() - kRotate4_;
         vPrime = kRotate7_ - tc_IdV9.triggerCellU();
       }
@@ -207,13 +207,13 @@ std::vector<uint32_t> HGCalCoarseTriggerCellMapping::getConstituentTriggerCells(
         int u = 0;
         int v = 0;
 
-        if (rocnum == 1) {
+        if (rocnum == kRoc0deg_) {
           u = uPrime;
           v = vPrime + u;
-        } else if (rocnum == 2) {
+        } else if (rocnum == kRoc120deg_) {
           u = uPrime + vPrime + 1;
           v = vPrime;
-        } else if (rocnum == 3) {
+        } else if (rocnum == kRoc240deg_) {
           u = kRotate7_ - vPrime;
           v = uPrime + kRotate4_;
         }

--- a/L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorCoarsenerImpl.cc
+++ b/L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorCoarsenerImpl.cc
@@ -37,7 +37,7 @@ void HGCalConcentratorCoarsenerImpl::coarsen(const std::vector<l1t::HGCalTrigger
     if (triggerTools_.isSilicon(tc.detId())) {
       thickness = triggerTools_.thicknessIndex(tc.detId(), true);
     } else if (triggerTools_.isScintillator(tc.detId())) {
-      thickness = 3;
+      thickness = kScintillatorPseudoThicknessIndex_;
     }
 
     if (fixedDataSizePerHGCROC_ && thickness == kHighDensityThickness_) {

--- a/L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorSuperTriggerCellImpl.cc
+++ b/L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorSuperTriggerCellImpl.cc
@@ -35,7 +35,7 @@ void HGCalConcentratorSuperTriggerCellImpl::createAllTriggerCells(
     if (triggerTools_.isSilicon(output_ids.at(0))) {
       thickness = triggerTools_.thicknessIndex(output_ids.at(0), true);
     } else if (triggerTools_.isScintillator(output_ids.at(0))) {
-      thickness = 3;
+      thickness = kScintillatorPseudoThicknessIndex_;
     }
 
     for (const auto& id : output_ids) {
@@ -104,9 +104,10 @@ void HGCalConcentratorSuperTriggerCellImpl::assignSuperTriggerCellEnergyAndPosit
                              ? double(kTriggerCellsForDivision_)
                              : double(superTCmapping_.getConstituentTriggerCells(stc.getSTCId()).size());
 
-    c.setHwPt(stc.getSumHwPt() / denominator);
-    c.setMipPt(stc.getSumMipPt() / denominator);
-    c.setPt(stc.getSumPt() / denominator);
+    double denominatorInv = 1. / denominator;
+    c.setHwPt(stc.getSumHwPt() * denominatorInv);
+    c.setMipPt(stc.getSumMipPt() * denominatorInv);
+    c.setPt(stc.getSumPt() * denominatorInv);
   } else if (energyDivisionType_ == oneBitFraction) {
     double frac = 0;
 
@@ -124,7 +125,7 @@ void HGCalConcentratorSuperTriggerCellImpl::assignSuperTriggerCellEnergyAndPosit
   if (triggerTools_.isSilicon(c.detId())) {
     thickness = triggerTools_.thicknessIndex(c.detId(), true);
   } else if (triggerTools_.isScintillator(c.detId())) {
-    thickness = 3;
+    thickness = kScintillatorPseudoThicknessIndex_;
   }
 
   GlobalPoint point;

